.github/workflows/codeql-codescan.yml

```diff
@@ -41,3 +41,6 @@ jobs:
 
        - name: Perform CodeQL Analysis
          uses: github/codeql-action/analyze@v2
+
+       - name: Upload coverage reports to Codecov
+         uses: codecov/codecov-action@v3
```

Comment:
Do not edit any files in the `.github` folder or any of its sub folders.

Remove these edits. They are duplicative. We already have code coverage calculations in another file.

Please follow my instructions

Reply:
@palisadoes Okay I've removed it.